examples/raw/upload.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Minimal example to upload a file via a HTML form
+ *
+ * Try it using the built-in webserver
+ * `ACCESS=youraccesstring php -S 0.0.0.0:8080`
+ * then visit http://localhost:8080/upload.php
+ */
+
+use Storj\Uplink\Uplink;
+
+ini_set('display_errors', 1);
+
+require '../../vendor/autoload.php';
+
+$access = Uplink::create()->parseAccess(getenv('ACCESS'));
+
+if ($_SERVER['REQUEST_METHOD'] === 'GET') {
+    echo <<<HTML
+        <form method="POST" enctype="multipart/form-data">
+            <input type="file" name="files[]" multiple />
+            <button>Submit</button>
+        </form>
+    HTML;
+    return;
+}
+
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+    $project = $access->openProject();
+    $project->ensureBucket('raw');
+
+    $files = $_FILES['files'] ?? [];
+    $nFiles = count($files['name'] ?? []);
+    for ($i = 0; $i < $nFiles; $i++) {
+        $upload = $project->uploadObject('raw', $files['name'][$i]);
+        $upload->writeFromResource(fopen($files['tmp_name'][$i], 'r'));
```

Comment:
Where does the `tmp_name` key come from?

Reply:
It is in the $_FILES superglobal. The file is created by PHP if the form has any `<input type="file">`.